### DISCUSSION TITLE
Difference between --mod --var-mod

### DISCUSSION THREAD

**Discussion by roberthardt** [vdemichev/DiaNN#766] [Q&A]
Dear Vadim,

I got a little confused regarding the setting up of additional modifications and hope you could clarify the issue? There are the options --var-mod  and --fixed-mod, which I thought should be enough. But there is also the parameter --mod. So my question now is, when to use the first two parameters and when the --mod instead?

Best

Robert

**Comment by vdemichev** [comment]
Hi Robert,

--mod is for spectral libraries primarily, to let DIA-NN know what's the mass of the modification. But yes, if you are searching with a spectral library, --var-mod and --fixed-mod by themselves are not necessary, but they will also do the job of --mod. If there are no errors which report unknown mods, you don't need --mod :)

Best,
Vadim
